packages/eslint-plugin/rules/api-no-slashes.js

```diff
@@ -1,6 +1,5 @@
-'use strict';
-const { getCallingIdentifier, getVariableInScope } = require('../utils/utils');
-const { get } = require('lodash');
+import { getCallingIdentifier, getVariableInScope } from '../utils/utils';
```

Comment:
I see that all the rule files (except `api-deprecation.ts`) are still `js` files, but the test files are renamed to `ts` files. Why is that?